transformations/numeric_to_word/numeric2word.py

```diff
@@ -0,0 +1,1025 @@
+import re
+import time
+import string 
+import calendar
+import datetime
+import numpy as np
+import phonenumbers
+from num2words import num2words
+from phonenumbers import carrier, timezone, geocoder
+
+special_numbers = ['911']
+
+math_sign = ['=', '<', '<=', '=>', '>', '!=', "**"]
+
+month_words = ["january", "jan", "jan.", "february", "feb", "feb.",
+               "march", "mar", "mar.", "april", "apr", "apr.",
+               "may", "june", "jun", "jun.", "july", "jul", "jul.",
+               "august", "aug", "aug.", "september", "sep", "sep.", 
+               "sept", "sept.", "october", "oct", "oct.", "november", 
+               "nov", "nov.", "december", "dec", "dec."]
+
+### Implementations 
+
+def recognize_transform(word, prev_word, next_word):
+    
+    # transforming by looking at previous or next words
+    if recognized_as_power_of_ten(word, prev_word):
+        remaining_word = word[2:]
+        remaining_word = '-' + remaining_word[1:] if remaining_word[0] == '−' else remaining_word
+        words = 'ten power ' + num2words(remaining_word)
+#         print('1', word, words)
```

Comment:
Please remove all the commented print statements, better not to have that in master.

Reply:
done, sorry for that